gvr-3dcursor/app/src/main/assets/settings.xml

```diff
@@ -226,11 +227,6 @@
             position="center"
             themeId="crystal_sphere"
             type="object">
-        <!--_VENDOR_TODO_ add your first device to the settings.xml here-->
```

Comment:
Simpler to uncomment this block and test the cursors than to play around with the existing settings. I've added the note regarding line 226 to the integration steps.

I've also changed the cursor names.